spec.html

```diff
@@ -33131,7 +33138,7 @@ <h1>
           1. Return _t_ - 𝔽(_offsetMs_).
         </emu-alg>
         <p>
-          Input _t_ is nominally a time value but may be any Number value.
+          Input _t_ is nominally a local-offset time value but may be any Number value.
           The algorithm must not limit _t_ to the time value range, so that inputs corresponding with a boundary of the time value range can be supported regardless of local UTC offset.
           For example, the maximum time value is 8.64 × 10<sup>15</sup>, corresponding with *"+275760-09-13T00:00:00Z"*.
           In an environment where the local time zone offset is ahead of UTC by 1 hour at that instant, it is represented by the larger input of 8.64 × 10<sup>15</sup> + 3.6 × 10<sup>6</sup>, corresponding with *"+275760-09-13T01:00:00+01:00"*.
```

Comment:
I suggest we redefine local-offset time value to account for this additional range, then type this AO properly.

Reply:
Alternatively, I don't think it'd be a problem to just expand the range of time value either, right?

Reply:
`UTC` is called with unbounded integral Number values, so we can't restrict the input to time values.

Reply:
Okay. I still think we should expand the range supported by local-offset time values.

Also, it's unrelated to this PR, but: `t` isn't *nominally* a time value, it's *nominally* a Number. It's "ostensibly" or "figuratively" a time value. I would update that term as well.

Reply:
I don't understand to which range local-offset time values should be extended? Any integral Number, including NaN (and possibly Infinity, too)? Because that's the range which can be produced by MakeDate. (And the output of MakeDate is passed as the input to UTC.)

Reply:
ISO 8601 timezone offsets permit ±24 hours (less 1 nanosecond). Time values are integers within the range -8,640,000,000,000,000 to 8,640,000,000,000,000 (100 million days on either side of the epoch). I'm suggesting that local time values have an extended range of ±8,640,000,086,400,000 (100,000,001 days on either side of the epoch).

Reply:
±8,640,000,086,400,000 suggests it's valid for a time zone offset to be ±24 hours, but just as you said, the maximum valid time zone offset is ±23:59:59.999.999.999 hours. Ignoring micro- and nanoseconds, we get ±23:59:59.999, which is ±86399999 = ±(86400000 - 1) milliseconds, so exactly the range already used within this PR.

Reply:
Can you point to where a local-offset time value is defined in that way in this PR? All I see is that it's restricted to the same range as un-offset time values (±8,640,000,000,000,000).

Reply:
Ah, I think I understand our misunderstanding.

The first commit has:
> An ECMAScript <dfn variants="local-offset time values">local-offset time value</dfn> is a time value adjusted by a time zone offset that is an integer number of milliseconds in the interval from -86,400,000 (exclusive) to 86,400,000 (exclusive).

I guess "time value adjusted by a time zone offset" can be read in two different ways:
1. Take a time value ±(8.64 * 10^15) and adjust it by ±86399999 to get ±8,640,000,086,399,999 as the final range.
2. "local-offset time value" is still a "time value", so restricted to ±(8.64 * 10^15), but additionally has some time zone offset applied.

Reply:
Yes, I read it the second way. I would prefer to explicitly state the final computed range in its definition.